pkg/reconciler/v1alpha1/testing/functional.go

```diff
@@ -440,6 +440,9 @@ func MarkRevisionReady(r *v1alpha1.Revision) {
 	r.Status.MarkContainerHealthy()
 }
 
+// TODO(josephburnett): rename KPAOption to PAOption because "kpa" and
+// "hpa" are specialized PA implementations. Rename is split across
+// knative/serving and knative/pkg repos so we must two it in two phases.
```

Comment:
How is knative/pkg involved?  That has no awareness of KPA (or it's a bug!).

Reply:
It's not.  I found https://github.com/knative/serving/blob/0f4b4d88eaccba3336a523f06e938823dc05b01a/pkg/reconciler/v1alpha1/testing/functional.go#L558 and made a mistake looking at the path.  Saw `github.com/knative/pkg/reconciler/v1alpha1/testing/functional.go` (sans `serving`) but it was `github.com/knative/serving/pkg/reconciler/v1alpha1/testing/functional.go`.

I'll make the updates to `functional.go` now in this PR.